doc/source/diagnostics/process_list.rst

```diff
@@ -42,6 +46,7 @@ INFORMATION_SCHEMA Tables
    :column ROWS_EXAMINED: The number of rows examined by the statement being executed (*NOTE:* This column is not updated for each examined row so it does not necessarily show an up-to-date value while the statement is executing. It only shows a correct value after the statement has completed.).
    :column ROWS_SENT: The number of rows sent by the statement being executed.
    :column ROWS_READ: The number of rows read by the statement being executed.
+   :column TID: The Linux Thread ID. This information provides the option to inspect or monitor the MySQL server threads with tools external to the server itself, such as Linux perf.
```

Comment:
Just Linux thread id, possibly with what it corresponds to in ps output, also how it's defined for threadpool.